contrib/setup-cms.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -e -o pipefail
+
+dep_dir="deps"
+install_dir=$(readlink -fm ${dep_dir}/install)
+
+[ ! -d "${install_dir}" ] && mkdir -p "${install_dir}"
+
+dep="cms"
+
+cd ${dep_dir}
```

Comment:
Please quote variable expansions. Maybe there's a shell linter we can apply to make sure it's all good.

Reply:
Done.

Scripts have been checked with https://github.com/koalaman/shellcheck (scripts were already formatted in a way that https://github.com/mvdan/sh made no changes).